bin/build_manylinux1_wheel.sh

```diff
@@ -24,6 +24,12 @@ then
     PIP=/opt/python/cp310-cp310/bin/pip
     CYTHON=/opt/python/cp310-cp310/bin/cython
     AUDITWHEEL=/opt/python/cp310-cp310/bin/auditwheel
+elif [ $PYTHON_VERSION == "3.11" ]
+then
+    PYTHON=/opt/python/cp311-cp311/bin/python
+    PIP=/opt/python/cp311-cp311/bin/pip
+    CYTHON=/opt/python/cp311-cp311/bin/cython
+    AUDITWHEEL=/opt/python/cp311-cp311/bin/auditwheel
```

Comment:
I guess that we can remove this https://github.com/emcache/emcache/pull/86/files#diff-6688e3e7b448091a9e4bb85ef4145d8f1f0f0d7f013ec6dee77300187f4bd80cR3 right?

Reply:
Oh. I think I can remove 3.7 branch. Thanks. :)

Reply:
@pfreixes Do you think should we can remove 3.11 branch? I think it'll be necessary.

Reply:
Why? arent you adding support for Python 3.11?

Reply:
@pfreixes

This process is required to build manylinux wheel. And It's branched with supported Python version.

Before this PR, we supported Python 3.7 to 3.10. After this PR, we supports 3.8 to 3.11. I forgot to remove if python version is 3.7, so I removed it.

Should I need to fix more?

Reply:
nops its ok